test-utils/src/main/kotlin/com/r3/bno/testing/SimpleMetaData.kt

```diff
@@ -0,0 +1,9 @@
+package com.r3.bno.testing
+
+import net.corda.core.serialization.CordaSerializable
+
+@CordaSerializable
+data class SimpleMembershipMetadata(val role: String = "", val displayedName: String = "")
+
+@CordaSerializable
+data class SomeCustomMembershipMetadata(val someCustomField: String)
```

Comment:
Can these classes be moved to https://github.com/corda/corda-solutions/tree/master/bn-apps/businessnetworks-test-utilities instead?

Reply:
will address in follow up PR. 

Slightly complicated by the fact that the AbstractFlow tests are within the same module.